lambdas/shared/t4_lambda_shared/preview.py

```diff
@@ -166,13 +169,14 @@ def extract_parquet(file_, as_html=True, skip_rows=False, *, max_bytes: int):
     # spare memory; part of the reason this is so inefficient: we've already read
     # the entire parquet file into a BytesIO by the time we get here
     if meta.num_row_groups:
-        # guess because we meta doesn't reveal how many rows in first group
+        # guess because meta doesn't reveal how many rows in first group
         num_rows_guess = math.ceil(meta.num_rows / meta.num_row_groups)
         size_guess = num_rows_guess * meta.num_columns * AVG_PARQUET_CELL_BYTES
         if skip_rows or (size_guess > get_available_memory()):
             # minimal dataframe with all columns and one row
             dataframe = pandas.DataFrame(columns=meta.schema.names)
             info['warnings'] = 'Large file: skipped rows to conserve memory, only showing column names'
```

Comment:
Where does this one end up? The file's metadata?

Reply:
It gets returned to the caller.